### PR TITLE
Invite Code System

### PR DESCRIPTION
**Pull request by LuisNeves05**

### CODE REVIEW

File: src/components/app.jsx [hunk 1]
```diff
@@ -1,10 +1,13 @@
-import React, { useEffect } from 'react';
+
+import React, { useState, useEffect, useRef } from 'react';
```
Comment (gmarquesetr) [inline src/components/app.jsx:2]: This useState and useRef are not used, right?

File: src/components/comments/styles.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ export default StyleSheet.create({
   commentModal: {
     justifyContent: 'flex-end',
     margin: 0,
-    paddingTop: Device.osName === 'iOS' ? 70 : 0,
+    paddingTop: Device.modelName === 'iOS' ? 70 : 0,
```
Comment (gmarquesetr) [inline src/components/comments/styles.js:8]: Why this change? According to [this documentation](https://docs.expo.dev/versions/latest/sdk/device/#devicemodelname) it can have different values than `iOS`

File: src/components/start/choose_login.jsx [hunk 3]
```diff
@@ -1,37 +1,48 @@
...
+  const { data: currentUser } = useGetCurrentUserQuery();
 
   const onSkipPress = async () => {
     setIsLoading(true);
     try {
       const response = await doLogin('guest@neftme.com', 'neftmeTest');
+
       if (response?.success) {
         await removeData('newUser');
+        console.log(currentUser);
```
Comment (gmarquesetr) [inline src/components/start/choose_login.jsx:26]: remove? :)

File: yarn-error.log
Comment (gmarquesetr) [inline yarn-error.log:1]: I think you can remove this file and maybe we can add it to .gitignore

File: src/components/start/choose_login.jsx (hunk 3 above)
Comment (LuisNeves05) [reply src/components/start/choose_login.jsx:26]: 😅

File: src/components/app.jsx (hunk 1 above)
Comment (LuisNeves05) [reply src/components/app.jsx]: Yup, after removing andaimes I forgot to clean up 🤣

File: src/components/global.ts [hunk 4]
```diff
@@ -3,6 +3,9 @@ import { LogBox } from 'react-native';
 import * as Device from 'expo-device';
 import TimeAgo from 'javascript-time-ago';
 import en from 'javascript-time-ago/locale/en';
+// import TaskManager from 'expo-task-manager';
+// import Constants from 'expo-constants';
+
```
[the remark this answers is not in the document]
Comment (LuisNeves05) [reply src/components/global.ts]: Done

File: src/components/start/choose_login.jsx (hunk 3 above)
Comment (LuisNeves05) [reply src/components/start/choose_login.jsx]: Done

File: src/components/app.jsx [hunk 5]
```diff
@@ -1,10 +1,13 @@
+
 import React, { useEffect } from 'react';
 import { Alert } from 'react-native';
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
 import { Provider } from 'react-redux';
 import * as Notifications from 'expo-notifications';
 import * as Device from 'expo-device';
+import Constants from 'expo-constants';
+import AsyncStorage from '@react-native-async-storage/async-storage';
```
Comment (gmarquesetr) [inline src/components/app.jsx:10]:
```suggestion
import React, { useEffect } from 'react';
import { Alert } from 'react-native';
import { NavigationContainer } from '@react-navigation/native';
import { createNativeStackNavigator } from '@react-navigation/native-stack';
import { Provider } from 'react-redux';
import * as Notifications from 'expo-notifications';
import * as Device from 'expo-device';
```
